apps/docs/src/app/core/component-docs/textarea/examples/textarea-state-example.component.html

```diff
@@ -4,12 +4,16 @@
     </label>
     <textarea fd-form-control id="textarea-51" placeholder="Field placeholder text"></textarea>
     <fd-form-message>Pellentesque metus lacus commodo eget justo ut rutrum varius nunc</fd-form-message>
+    <br/>
 </div>
 <div fd-form-item>
     <label fd-form-label for="textarea-52">
         Valid Textarea
     </label>
     <textarea fd-form-control id="textarea-52" placeholder="Field placeholder text" [state]="'valid'"></textarea>
+    <fd-form-message [type]="'success'">Pellentesque metus lacus commodo eget justo ut rutrum varius nunc
+    </fd-form-message>
+    <br/>
```

Comment:
same here br tags not needed

Reply:
It just looks weird without it, it's example.